yt_dlp/utils/traversal.py

```diff
@@ -54,6 +71,7 @@ def traverse_obj(
                             Read as: `{key: traverse_obj(obj, path) for key, path in dct.items()}`.
         - `any`-builtin:    Take the first matching object and return it, resetting branching.
         - `all`-builtin:    Take all matching objects and return them as a list, resetting branching.
+        - `filter`-builtin: Return the value if it is truthy, `None` otherwise.
```

Comment:
Should there be some documentation here or elsewhere that lists the non-built-in helpers `require()`, `find_element[s]()` with examples? If not here, in the developer section of the manual. Or link to explanatory text in this PR (simple, but not a good long term solution).

Reply:
Yes, developer markdown, but that part will have to be overhauled a bit anyways